build/build.proj

```diff
@@ -22,17 +22,22 @@
     <VsManProjectFile Include="$(RepositoryRootDirectory)src\VsixV3\ProjectSystemPackage\Microsoft.VisualStudio.ProjectSystem.Managed.vsmanproj" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <!-- NuGet will not automatically find the correct version of MSBuild. We need to specify the path. -->
+    <NugetArguments>-verbosity quiet -msbuildpath &quot;$(DevenvDir)\..\..\MSBuild\15.0\bin&quot;</NugetArguments>
```

Comment:
It's incomplete work - NuGet doesn't yet enumerate the installed Dev15 versions of msbuild and use one.  Even still, I'd rather be explicit about it and specify which one to use.